conda_pack/core.py

```diff
@@ -771,19 +775,6 @@ def load_environment(prefix, on_missing_cache='warn'):
     return files
 
 
-def strip_prefix(data, prefix, placeholder=PREFIX_PLACEHOLDER):
```

Comment:
Not needed because we've inlined the try/except approach to binary file replacement below.

Reply:
The thing is we're not actually stripping things out anymore. We're simply detecting whether or not the file is binary and acting accordingly. The replacement is done by other code; this was a redundancy.

I can certainly create a function that determines text/binary.